helios-client/src/main/java/com/spotify/helios/common/descriptors/DeploymentGroupTasks.java

```diff
@@ -115,6 +118,18 @@ public String toString() {
            + '}';
   }
 
+  /**
+   * Returns a string of an instance of this class that gives a local view of the
+   * {@link RolloutTask}s based on the taskIndex. Useful if there are many {@link RolloutTask}s.
+   */
+  public String toSummaryString() {
```

Comment:
I'd opt to make this be the toString implementation instead - do we ever really want a task list of 100s of items returned as a String?